triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -130,12 +130,12 @@ class Server(dar: Option[Dar[(PackageId, Package)]], triggerDao: Option[TriggerD
     }
   }
 
-  private def listRunningTriggers(jwt: Jwt): Either[String, Vector[UUID]] = {
+  private def listRunningTriggers(token: String): Either[String, Vector[UUID]] = {
```

Comment:
With the changes I suggested above, these functions would then take a username string as parameter.

Reply:
Yeah. Again I'm not convinced this is a particularly useful change. I think it's nicer to just have the (encoded) pair at our disposal and decode/decrypt for a rendering of the party where we need to. Not against validating the user component up-front as you suggest earlier.

Reply:
resolved by the preceding discussion - sticking with encrypted tokens, no extra maps.